Nos/Views/Discover/DiscoverGrid.swift

```diff
@@ -0,0 +1,135 @@
+                                        router.push(author)
+                                    }
+                                    .padding(.horizontal, 13)
+                                    .padding(.top, 5)
+                                    .readabilityPadding()
+                                    .task {
+                                        // TODO: optimize. Probably only needed once per author.
+                                        subscriptions[author.id] =
+                                            await relayService.requestMetadata(
+                                                for: author.hexadecimalPublicKey,
+                                                since: author.lastUpdatedMetadata
+                                            )
+                                    }
+                                }
+                            }
+                            .padding(.bottom, 16)
+                        }
+                        .doubleTapToPop(tab: .discover) { proxy in
+                            if let firstAuthor = authors.first {
+                                proxy.scrollTo(firstAuthor.id)
+                            }
+                        }
+                    case .empty:
+                        EmptyView()
+                    case .loading, .stillLoading:
+                        FullscreenProgressView(
+                            isPresented: .constant(true),
+                            text: searchController.state == .stillLoading ?
+                            String(localized: .localizable.notFindingResults) : nil
+                        )
+                    case .results:
+                        ScrollView {
+                            LazyVStack {
+                                ForEach(searchController.authorResults) { author in
+                                    AuthorCard(author: author) {
+                                        router.push(author)
+                                    }
+                                    .padding(.horizontal, 15)
+                                    .padding(.top, 10)
+                                    .readabilityPadding()
+                                }
+                            }
+                            .padding(.top, 5)
+                        }
+                        .doubleTapToPop(tab: .discover) { proxy in
+                            if let firstAuthor = searchController.authorResults.first {
+                                proxy.scrollTo(firstAuthor.id)
+                            }
+                        }
+                    }
+                }
+                .preference(key: SizePreferenceKey.self, value: geometry.size)
+            }
+        }
+    }
+
+    var scrollingPicker: some View {
+        ScrollView(.horizontal) {
+            HStack(spacing: 2) {
+                ForEach(FeaturedAuthorCategory.allCases, id: \.self) { category in
+                    Button(action: {
```

Comment:
These buttons are way too small, they are hard for me to click with my mouse much less a thumb. In the designs the full bar has a height of 38px. Apple's recommended minimum is 40, so I suggest we give the HStack here a minHeight of 38px and then make sure the button tap targets fill that height.

Reply:
I think the text should be bigger too. These things could be addressed in a future PR or ticket but I think this is actually unusable for people with big fingers.

Reply:
I made the text bigger and set min width and height for the button.